src/Tribe/Taxonomy/Taxonomy_Provider.php

```diff
@@ -53,4 +54,25 @@ protected function add_filters() {
 	public function event_tag_actions( $actions, WP_Term $tag ) {
 		return $this->container->make( Event_Tag::class )->event_tag_actions( $actions, $tag );
 	}
+
+	/**
+	 * Modifies the tag slug for the post_tag taxonomy to include an "events" prefix.
+	 *
+	 * @since TBD
+	 * 
+	 * @param array  $args The taxonomy arguments.
+	 * @param string $taxonomy The name of the taxonomy.
+	 * 
+	 * @return array The modified taxonomy arguments.
+	 */
+	public function modify_tag_taxonomy_args( $args, $taxonomy ) {
+		if ( 'post_tag' === $taxonomy ) {
+			$args['rewrite'] = [
+				'slug'       => 'events/tag',
+				'with_front' => false,
+			];
+		}
+
+		return $args;
+	}
```

Comment:
This should not go here - the logic does not belong in the Service Provider.
Instead, return 
`$this->container->make( Event_Tag::class )-> modify_tag_taxonomy_args( $args, $taxonomy );`

and put the logic in the `Event_Tag` class.